### ISSUE TITLE
508-defect-2 [FOCUS]: Back to top button's focus should be placed on the `main` or `h1` upon activation

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#25018]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Josh

## User Story or Problem Statement

As a keyboard user, I expect focus to be placed on the `main` or `h1` so I don't have to tab through the navigation all over again.

As a screen reader user, I expect focus to be placed on the `main` or `h1` so my screen reader can announce to me that I've returned to the top of the page.

## Details

https://user-images.githubusercontent.com/14154792/119006514-73745400-b95e-11eb-9c5f-60d26f3a26ec.mov

Focus is lost upon activating the back to top button.

## Acceptance Criteria

- [ ] Focus is placed on the `h1` of the page using `tab-index="-1"`

## Steps to Recreate

1. Turn on a screen reader
2. Tab to the back to top button
3. Activate it and confirm no announcement is made

**Comment by laflannery** [comment]
This looks resolved - Tested this on a couple pages on desktop and mobile and focus goes to the H1.  Will close in 1 week unless told otherwise
